vc-defer.el

```diff
@@ -309,10 +309,9 @@ global auto reverts back on, use \\[global-auto-revert-mode]."
     ;; are very intrusive in the trace output.
     (global-auto-revert-mode 0)
     (dolist (buffer (buffer-list))
-      (if auto-revert-mode
-          (auto-revert-mode -1))
-      (if auto-revert-tail-mode
-          (auto-revert-tail-mode -1)))
+      (with-current-buffer buffer
+        (auto-revert-mode -1)
```

Comment:
nit: above you're using the argument `0` to disable `global-auto-revert-mode`. Any nonpositive number works, but it would be nice to stay consistent.